File: src/com/app/test/game/source/AAnswerType.java
package com.app.test.game.source;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({AnswerType.NONE
        , AnswerType.NORMAL
        , AnswerType.PROVER})
@Retention(RetentionPolicy.SOURCE)
public @interface AAnswerType {
}

File: app/src/main/java/com/app/test/game/source/AAnswerType.kt
package com.app.test.game.source

import androidx.annotation.IntDef
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

@IntDef(AnswerType.NONE, AnswerType.NORMAL, AnswerType.PROVER)
@Retention(RetentionPolicy.SOURCE)
annotation class AAnswerType